cdap-data-fabric/src/main/java/io/cdap/cdap/data2/metadata/dataset/MetadataDataset.java

```diff
@@ -833,10 +841,16 @@ private SearchResults searchByCustomIndex(SearchRequest request) throws BadReque
     Set<EntityScope> entityScopes = searchRequest.getEntityScopes();
     List<SearchTerm> searchTerms = new LinkedList<>();
     Consumer<String> termAdder = determineSearchFields(namespace, entityScopes, searchTerms);
-    String searchQuery = searchRequest.getQuery();
+
+    /**String searchQuery = searchRequest.getQuery();
```

Comment:
Remove the code instead of commenting it, git will show it in the diff

Reply:
Addressed in [#11489](https://github.com/cdapio/cdap/pull/11489)